### ISSUE TITLE
Bug :: PageService :: before

### ISSUE THREAD

**Issue by AhmedKamal20** [egyptian-geeks/website#10]
We depend on before(created_at) in `PageService` to determine where we will start the next page
but the thing is, from what i noticed, it's not that accurate
example: 

```
$ User.order(created_at: :desc).limit(22)
[ ........
#<User:0x000055e86e6e1368
  id: 12049,
  name: "Doaa El-Saeed",
  fid: "1986253641700930",
  created_at: Wed, 07 Feb 2018 12:51:39 UTC +00:00,
  updated_at: Wed, 07 Feb 2018 12:51:39 UTC +00:00>,
 #<User:0x000055e86e6cded0
  id: 12048,
  name: "Ahmad Abd El-Aal",
  fid: "1994352697492691",
  created_at: Wed, 07 Feb 2018 12:51:39 UTC +00:00,
  updated_at: Wed, 07 Feb 2018 12:51:39 UTC +00:00>,
 #<User:0x000055e86e6cd548
  id: 12047,
  name: "Mackenzie Child",
  fid: "10159874449045043",
  created_at: Wed, 07 Feb 2018 12:50:56 UTC +00:00,
  updated_at: Wed, 07 Feb 2018 12:50:56 UTC +00:00>]
```
12049 & 12048 have the same created_at so in my server 12048 dropped from the index and page 2 started from 12047 instead of 12048

and in Posts Index, first post in every page is the last post from the previous page
you can see it here : http://egygeeks.emadelsaid.com/

**Comment by emad-elsaid** [comment]
hummm, how can we fix that 😄

**Comment by AhmedKamal20** [comment]
I understand the first point , but the second point with posts is weird

**Comment by AhmedKamal20** [comment]
OK, i guess i figured out what cause the second point : 
http://egygeeks.emadelsaid.com/posts?before=2018-01-27+07%3A50%3A48+UTC
If you remove `+UTC` from the URL it works fine, So, i think will be better to pass `before` to the query as `Time` not `String`

**Comment by AhmedKamal20** [comment]
Fixed in #11 , By Changing how Pagination works
